src/world/area_hos/hos.h

```diff
@@ -0,0 +1,25 @@
+#ifndef _WORLD_AREA_HOS_HOS_H_
+#define _WORLD_AREA_HOS_HOS_H_
+
+enum {
+    AB_HOS_0       = AreaByte(0),
+    AB_HOS_1       = AreaByte(1),
+    AB_HOS_2       = AreaByte(2),
+    AB_HOS_3       = AreaByte(3),
+    AB_HOS_4       = AreaByte(4),
+    AB_HOS_5       = AreaByte(5),
+    AB_HOS_6       = AreaByte(6),
+    AB_HOS_7       = AreaByte(7),
+    AB_HOS_8       = AreaByte(8),
+    AB_HOS_9       = AreaByte(9),
+    AB_HOS_A       = AreaByte(10),
+    AB_HOS_B       = AreaByte(11),
+    AB_HOS_C       = AreaByte(12),
+    AB_HOS_D       = AreaByte(13),
+    AB_HOS_E       = AreaByte(14),
+    AB_HOS_F       = AreaByte(15),
+};
+
+#define AF_HOS(index) AreaFlag(index)
```

Comment:
What's the point of these macros? Why not just use AreaFlag(index)? Is it to make area-specific code not compile if you blindly copy between areas?

Reply:
The purpose is to disambiguate AreaFlags from different areas, which will allow us to make an enum in the area header and give them unique names in the future. E.g., AreaFlag(1) for DGB becomes AF_DGB(1), which we can rename AF_DGB_CaughtBySentinel. I didn't per-emptively make a enum for the flags because there's 0x60 flags and most of them go unused.